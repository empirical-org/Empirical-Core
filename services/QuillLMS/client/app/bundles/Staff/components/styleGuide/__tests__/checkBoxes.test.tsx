import * as React from "react";
import { render } from "@testing-library/react";

import Checkboxes from "../checkBoxes";

describe('Checkboxes', () => {
  test('it should render', () => {
    const { asFragment } = render(<Checkboxes />);
    expect(asFragment()).toMatchSnapshot();
  })
})
