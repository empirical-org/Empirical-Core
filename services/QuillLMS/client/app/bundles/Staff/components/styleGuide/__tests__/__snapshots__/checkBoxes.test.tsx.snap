// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkboxes it should render 1`] = `
<DocumentFragment>
  <div
    id="checkboxes"
  >
    <h2
      class="style-guide-h2"
    >
      Checkboxes
    </h2>
    <div
      class="states-and-modes-container"
    >
      <h4
        class="style-guide-h4"
      >
        States
      </h4>
      <div
        class="options-container"
      >
        <div
          class="option-container"
        >
          <p
            class="option-label"
          >
            Inactive
          </p>
          <div
            class="checkbox-container"
          >
            <button
              class="quill-checkbox unselected light"
              role="checkbox"
              type="button"
            />
            <span
              class=" light"
            >
              Text
            </span>
          </div>
        </div>
        <div
          class="option-container"
        >
          <p
            class="option-label"
          >
            Active
          </p>
          <div
            class="checkbox-container"
          >
            <button
              class="quill-checkbox selected light"
              role="checkbox"
              type="button"
            >
              <img
                alt="check"
                src="https://assets.quill.org/images/icons/xs/check-small-white.svg"
              />
            </button>
            <span
              class=" light"
            >
              Text
            </span>
          </div>
        </div>
        <div
          class="option-container"
        >
          <p
            class="option-label"
          >
            Indeterminate
          </p>
          <div
            class="checkbox-container"
          >
            <button
              class="quill-checkbox indeterminate light"
              role="checkbox"
              type="button"
            >
              <img
                alt="dash"
                src="https://assets.quill.org/images/icons/xs/dash-white.svg"
              />
            </button>
            <span
              class=" light"
            >
              Text
            </span>
          </div>
        </div>
        <div
          class="option-container"
        >
          <p
            class="option-label"
          >
            Disabled
          </p>
          <div
            class="checkbox-container"
          >
            <button
              class="quill-checkbox unselected light disabled"
              role="checkbox"
              type="button"
            />
            <span
              class="disabled light"
            >
              Text
            </span>
          </div>
        </div>
      </div>
      <h4
        class="style-guide-h4"
      >
        Modes
      </h4>
      <div
        class="options-container"
      >
        <div
          class="option-container"
        >
          <p
            class="option-label"
          >
            On Light
          </p>
          <div
            class="checkboxes"
          >
            <div
              class="checkbox-container"
            >
              <button
                class="quill-checkbox unselected light"
                role="checkbox"
                type="button"
              />
              <span
                class=" light"
              >
                Text
              </span>
            </div>
            <div
              class="checkbox-container"
            >
              <button
                class="quill-checkbox selected light"
                role="checkbox"
                type="button"
              >
                <img
                  alt="check"
                  src="https://assets.quill.org/images/icons/xs/check-small-white.svg"
                />
              </button>
              <span
                class=" light"
              >
                Text
              </span>
            </div>
          </div>
        </div>
        <div
          class="option-container dark"
        >
          <p
            class="option-label"
          >
            On Dark
          </p>
          <div
            class="checkboxes"
          >
            <div
              class="checkbox-container"
            >
              <button
                class="quill-checkbox unselected dark"
                role="checkbox"
                type="button"
              />
              <span
                class=" dark"
              >
                Text
              </span>
            </div>
            <div
              class="checkbox-container"
            >
              <button
                class="quill-checkbox selected dark"
                role="checkbox"
                type="button"
              >
                <img
                  alt="check"
                  src="https://assets.quill.org/images/icons/xs/check-small.svg"
                />
              </button>
              <span
                class=" dark"
              >
                Text
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
